---
title: "Detailed Methods for High Dimensional Mediation for Massively Polygenic Traits"
author: J. Matthew Mahoney and Anna L Tyler
date: '`r format(Sys.Date(), "%B %d, %Y")`'
output: 
  html_document:
    code_folding: hide
    collapsed: no
    toc: yes
    toc_float: yes
---


## Diversity Outbred Mice

## Trait measurements

## Trait selection

We filtered the measured traits in this study to a set of 
relatively non-redundant measures that were well-represented
in the population (having at least XXX individuals), and spanned 
multiple aspects of metabolic disease. A complete description of
trait filtering can be found in File XXX (1b.Trait_Selection.Rmd).

We took two approaches for traits with multiple redundant measurements,
for example logintudinal body weights. In the case of longitudinal 
measurements, we used the final measurement, as this was the closest
physiological measurement to the measurement of gene expression, which
was done at the end of the experiment. The labels for these traits are 
have the word "Final" appended to their name. For traits with multiple 
highly related measurements, such as cholesterol, we used the first
principal component of the group of measurements. For example, we used 
the first principal component of all LDL measurements as the measurement 
of LDL. For each set of traits, we ensured the first principal component 
had the correct sign by correlating it with the average of the traits. 
For correlation coefficients (R) less than 0, we multiplied the principal 
component by -1. The labels for these traits have the term "PC1" appended 
to their name.

## RNA Sequencing

RNA sequencing was performed as described earlier [cite]. Briefly,...

## Processing of RNA sequencing data

We used the Expectation-Maximization algorithm for Allele Specific Expression 
(EMASE) [cite] to quantify multi-parent allele-specific and total expression from 
RNA-seq data for each tissue. EMASE was performed by the Genotype by RNA-seq 
(GBRS) software package (https://gbrs.readthedocs.io/en/latest/). In the 
process, R1 and R2 FASTQ files were combined and aligned to a hybridized (8-way) 
transcriptome generated for the 8 DO founder strains as single-ended reads. GBRS 
was also used to reconstruct the mouse genotype probabilities along ~69K markers, 
which was used for confirming genotypes in the quality control (QC) process.
For the QC process, we used a Euclidean distances method (developed by Greg 
Keele - Churchill Lab) to compare the GBRS genotype probabilities between the 
tissues and the genotype probabilities array for all mice. The counts matrix for 
each tissue was processed to filter out transcripts with less than one read for 
at least half of the samples. RNA-seq batch effects were removed by regressing 
out batch as a random effect and considering sex and generation as fixed effects 
using lme4 R package. RNA-Seq counts were normalized relative to total read counts 
using the variance stabilizing transform (VST) as implemented in DESeq2 and using 
rank normal score.

We used R/qtl2 [cite] to perform eQTL analysis. We used the rank normal score 
data and used sex and DO generation as additive covariates. We also used kinship
as a random effect. We used permutations to find a LOD threshold of 8 for 
significant QTLs which corresponded to a $p$ value of 0.05. 

To assess whether eQTL were shared across tissues, we compared eQTLs for 
each transcript across tissues. Significant eQTLs within 4Mb of each other 
were considered overlapping. We considered local and distal eQTLs separately.

To estimate local and distal heritability of each transcript, we scaled each
normalized transcript to have a variance of 1. We then modeled this transcript
with the local genotype using the fit1() function in R/qtl with a kinship 
correction. We used the resulting model to predict the transcript values. 
The variance of the predicted transcript its local heribatility. We then 
estimated the heritability of the residual of the model fit. The variance
of the residual multiplied by its heritability is the distal heritability 
of the transcript. 

We compared local and distal estimates of heritability to measures of trait
relevance for each transcript. Trait relevance, was the Pearson correlation
(R) between the transcript and the trait.

## Kernelization

Before running high-dimensional mediation analysis, we kernelized the 
genotype, transcriptomic, and phenotype data sets to generate $n \times n$
matrices in where $n$ is the number of individual mice. Each matrix described
the relationships among individuals based on their genome, transcriptome, or 
phenome. Each matrix was generated as follows:

### Kernelizing the genome

The kernel matrix of the genome is the overall kinship matrix as 
calculated by calc_kinship() in the R package qtl2 [cite]. We further 
mean-centered this matrix based on DO generation. 

### Kernelizing the transcriptome

Prior to kernelizing the transcriptome, we regressed out the
effects of sex and DO generation. We then mean centered and 
standardized transcripts across individuals. The kernel matrix
($K_t$) for the transcriptome of each tissue was calculated 
as follows:

\begin{equation*}
K_t = \frac{Tr \times Tr^T}{n_{Tx}}
\end{equation}

where $Tr$ is a matrix of transcript abundances with individuals 
in rows and transcripts in columns, $Tr^T$ is $Tr$ transpose,
and $n_{Tx}$ is the number of transcripts in the matrix.
We kernelized each tissue's transcriptome and then averaged
across all tissues to generate a single transcriptome kernel
for all tissues.

### Kernelizing the phenome

The phenome kernel was constructed the same way as the transcriptome
kernel. We regressed out sex and DO generation and then mean centered
and standardized the phenotypes. We used knn.impute() [cite] to impute
missing values. We then used the above equation to generate the phenome 
kernel replacing the trancript matrix and number
of transcripts with the phenotype matrix and number of phenotypes.

## High-Dimensional Mediation Analysis

Matt has text for this section.

## Calculation of Loadings

## TWAS in DO mice
We performed a transcriptome-wide analysis (TWAS) [cite] in the 
DO mice to compare to the results of high-dimensional mediation. 
To perform TWAS, we fit a linear model to explain variation in 
each transcript across the population using the genotype at the 
nearest marker to the gene transcription start site (TSS). We
used kinship as a random effect and sex, diet, and DO generation 
as fixed effects. The predicted transcript from each of these models
was the imputed transcript based only on the local genotype.

**model equation**

We correlated each imputed transcript with each of the metabolic
phenotypes after adjusting phenotypes for sex, diet, and DO generation.
To calculate significance of these correlations, we performed permutation 
testing by shuffling labels of individual mice and recalculating 
correlation values. Significant correlations were those more extreme
than any of the permuted values, corresponding to an empirical $p$
value of 0. These are transcripts whose locally encoded expression
level was significantly correlated with one of the metabolic traits.
This suggests an association between the genetically encoded transcript
level and the trait, but does not identify a direction of causation. 



## Literature support for genes
To determine whether each gene among those with large loadings or
large heritability had a supported connection to obesity or diabetes
in the literature, we used the R package easyPubMed [cite]. We searched
for the terms ("diabetes" OR "obesity") along with the tissue name (adipose,
islet, liver, or muscle), and the gene name. We restricted the gene name
to appear in the title or abstract as some short names appeared at random
in contact information. We checked each gene with apparent literature
support by hand to verify that support, and we removed spurious associations.
For example, FAU is used as an acronym for fatty acid uptake and CAD is 
used as an acronym for coronary artery disease. Both terms co-occur with 
the terms diabetes and obesity in a manner independent of the genes 
\textit{Fau} and \textit{Cad}. Other genes that co-occurred with
diabetes and obesity, but not as a functional connection were similarly
removed. For example, the gene \textit{Rpl27} is used as a reference gene 
for quantification of the expression of other genes, and co-occurrence with
diabetes and obesity is a coincidence. We counted the abstracts associated
with diabetes or obesity and each gene name, and determined that a gene
had literature support when it had at least two abstracts linking it to
the terms diabetes or obesity in the respective tissue. 

## CC-RIX mice

## CC-RIX genotypes
We used the most recent common ancestor (MRCA) genotypes for the 
Collaborative Cross (CC) mice available on the University of 
North Carolina website: 
\url{http://www.csbio.unc.edu/CCstatus/CCGenomes/}

To generate CC-RIX genotypes, we averaged the haplotype 
probabilities for the two parental strains at each locus. 

## Imputation of gene expression in CC-RIX

To impute gene expression in the CC-RIX, we performed the following
steps for each transcript in each tissue (adipose, liver, and skeletal muscle):
  1. Calculate diploid CC-RIX genotype for all CC-RIX individuals
    at the marker nearest the transcription start site of the transcript.
  2. Multiply the genotype probabilities by the eQTL coefficients
    identified in the DO population. 

To check the accuracy of the imputation, we correlated the 
each imputed transcript with the measure transcript. The
average Pearson correlation (r) was close to 0.5 for all 
three tissues (Supp. Fig. XXXA), and as expected, the 
correlation between the imputed transcript and the measured
transcript was highly positively dependent on the local 
eQTL LOD score of the transcript (Supp. Fig. XXXB).

## Prediction of CC-RIX traits

We used both measured expression and imputed expression
combined with the results from HDMA in the to predict
phenotype in the CC-RIX. The traits measured in the DO
and the CC-RIX were not identical, so we limited our
prediction to body weight, which was measured in both
populations, and was the largest contributor to the 
phenotype score in the DO. 

For each CC-RIX individual, we multiplied the transcript
abundaces across the transcriptome by the loadings derived
from the HDMA in the DO population (Fig. XXXA). This resulted 
in a vector with $n$ elements, where $n$ is the number of 
transcripts in the trancriptome. Each element was a weigted 
value that combined the relative abundance of the transcript 
with how that abundance affected the phenotype. We averaged 
the values in this vector to calculate an overall predicted
phenotype score for the individual CC-RIX animal. 

After calculating this predicted phenotype value across
all CC-RIX animals, we correlated the predicted values from
each tissue with measured body weight (Fig. XXXB).