---
title: "Excised"
author: Anna
date: '`r format(Sys.Date(), "%B %d, %Y")`'
output: 
  pdf_document
---



Tissue-derived transcriptomes are increasingly viewed 
as a biological bridge between genetic risk factors for 
complex disease and their associated pathophysiology. 
Gene expression quantitative trait loci (eQTL) and 
large-scale genome-wide association (GWAS) facilitate 
a conceptual model to potentially infer genetic variants 
that alter nearby gene expression and account for 
co-localized disease risk. However, these studies in humans 
are consistently limited by mismatched populations and sample 
sizes for eQTL vs GWAS, limited tissue availability, and 
human population history and structure. Here, we perform a 
multi-tissue analysis of multiple traits related to metabolic 
disorders in an outbred population of laboratory mice. The 
combination of comprehensive clinical phenotyping with 
corresponding tissue gene expression from the same genetically 
diverse animals enables an advanced analytical strategy linking 
genetics, tissue-specific transcriptomes, and disease biomarkers. 
We infer paths of QTL that alter gene expression patterns linked 
to correlated metabolic traits, thereby augmenting genetic 
associations with pathway alterations that reveal the underlying 
molecular dysfunction of disease and candidate proteins for 
intervention.



drive to associate genetic variation with variation in risk of complex disease
	GWAS, TWAS

missing heritability

omnigenic model

complex systems - 
	biological systems are complex at every level of organization
		low heritability genes account for most of expression-mediated disease heritability
			in both humans and mice, across multiple, relevant tissues.
		traits are canalized against variation in genes and environment
			transcripts that vary widely in abundance based on cis effects cannot have a large
			effect on traits
			transcripts that have a large effect on traits cannot vary widely in abundance based on 
			local effects
			both scenarios create a fragile network
		biological networks are robust to perturbation - canalized
		and yet biological networks are also evolvable: there is variation in these newtorks across
			individuals that does have an effect on traits
		how do we associate the variability in gene expression with variability in traits?
			need to fit the whole model - trait-correlated transcripts have low heritability
				locally driven transcripts have low trait correlation
		need a holistic approach - in a complex disease the effects of individual variants and
			fluxuation of individual transcripts do not renormalize at higher levels of 
			organization
		cannot trace this small-scale fluctuation to fluctuation in traits, except in Mendelian disease
			rather we need to look at higher order structure
		interestingly, the extremely high-dimensional variation at the lower levels is transmitted
			to lower dimensional fluctuation at higher levels (true? cite? how to explain?)
			or is it that the variation at all levels is low-dimensional? 
			or that a few dimensions of the variation at lower levels translates to variation at the 
				higher levels?
			 
	emergence with low-dimensionality
		how many axes of variation are there? 
		top 10 PCs of adipose expression matrix explain: 16.43, 6.55, 4.24, 3.04, 2.44, 
			1.95, 1.28, 1.23, 1.14, and 1.04% of variation
		relatively high-dimensional
		top 10 PCs of phenotype matrix explain: 58.12, 11.11, 9.87, 4.41, 3.75, 2.77, 2.24, 
			2.06, 1.31, and 0.82% of variation
		many fewer axes of variation (but also way fewer things measured)
	
	individuals are individuals
	
	mediated genetic effects for each transcript give detailed estimate of causal effects
		but global signature may be more useful for identifying potential targets through CMAP
		or TRiAGE - does TRiAGE agree with CMAP?
		not really. Btk is ranked 88, Ikbkb is ranked 2454, Ikbke is ranked 4715
		

Mediation analysis is well suited to identification of causal genetic 
variants because we know that the arrow of causation extends from 
gene to gene expression and phenotypes and there is no causal arrow
backward from gene expression and phenotype to genotype. Thus there 
is number of possible models fitting the data is greatly reduced (bmediatR: 35533209).


## Notes from literature
allelic heterogeneity is a problem for mapping and dilutes statistical 
power to identify disease-associated loci [11404818] - because our method
doe not use individual markers for mapping, we avoid this issue. we aim 
at variation in gene expression levels. This readout avoids issues with
mapping resolution and is gene-based. If the overall point of mapping is 
to identify individual genes and molecular mechanisms, then we can actually
get at this more directly by looking at variation in levels of gene products
rather at the genome itself, and then inferring the effect on gene products (?)

If disease variants largely are regulatory in nature, lets look at the output
of that regulation (we are only looking at abundance, though, not splicing.
Pritchard has a paper [27126046] that says that variants that affect splicing,
not expression levels, are major contributors to complex traits: trait-associated
SNPs were more enriched in splicing eQTL, than in abundance eQTL.)
Also, the genome sequence is constant across cell types, whereas gene expression
is cell-type-specific, so you need to measure lots of it in each individual

## Abstract/Introduction

The transcriptome is increasingly viewed as a bridge between genetic 
risk factors for complex disease and their associated pathophysiology.
Powerful insights into disease mechanism can be made by linking genetic
variants affecting gene expression (expression quantiatitive trait loci - 
eQTLs) to phenotypes. 

**Make the case for looking at the distally
heritable signature here. Not as a failure of local eQTL, but just as
another promising avenue of exploration**.

In the end we show that there is a distally heritable signature that
(a) is related to disease, (b) translates across populations and species, 
and (c) is targetable. Although local eQTLs give us promise of understanding
very specific mechanisms and targeting very specific things, distally
heritable signatures give us insights into mechanism at an extrapolated 
level - inflammation, metabolism - and they are targetable based on the work 
of CMAP. A fruitful avenue of exploration for understanding and treating 
common diseases.

Studies have shown that the majority of 
expression-mediated disease heritability is explained by transcripts with 
low local heritability. 

Thus it is possible that there are identifiable, distally heritable 
transcriptomic signatures that mediate the effect of transcription on 
phenotype. We propose high-dimensional mediation (HDM) to identify the 
distally heritable portion of the transcriptome that mediates the effect 
of genome on phenome. (a bit more about the technique) We performed 
HDM in a population of 371 diversity outbred (DO) mice, both male and 
female, maintained on a high-fat, high-sugar diet. (DO mice are...) 
The mice were extensively phenotyped for multiple, clinically relevant 
metabolic measures including measures of obesity and insulin resistance. 
We measured transcriptome-wide gene expression in four tissues: white 
adipose tissue, pancreatic islets, liver, and skeletal muscle. We performed 
HDM using kernelized matrices of the genome, transcriptomes, and phenome 
to identify genome, transcriptome, and phenome scores where the 
transcriptome score perfectly mediated the effect of the genome score 
on the phenome score. We calculated loadings onto individual
transcripts and phenotypes to interpret the scores. Using 
loadings on individual transcripts, we were able to functionally annotate
heritable transcriptional programs that lead to increased weight gain
and insulin resistance in the DO population. We combined transcript loadings 
with local eQTL allele effects to identify risk variants and protective 
variants in the DO mice. These annotations of risk and protective alleles
allowed us to perform a TWAS-like analysis within the DO population as well 
as in an independent population of Collaborative Cross (CC) F1 mice (CC-RIX). 
In the DO population, each mouse carried a relatively balanced set of risk 
and protective alleles. The mean of these local allele effects (the 
polygenic risk score - PGS) explained between 14\% (skeletal muscle eQTL) 
and 47\% (adipose eQTL) of trait variance in the DO population, mirroring 
success of TWAS within European populations. However, when we calculated 
PRS in the CC-RIX based on the values derived in the DO population, PRS 
completely failed to predict phenotype (numbers forthcoming). Although 
all the alleles in the CC-RIX are all present in the DO population, 
the presence of risk and protective alleles as defined in the DO did not 
predict outcomes in the CC-RIX, supporting the hypothesis that local 
eQTLs contribute minimally to phenotypic outcomes. We performed an analogous 
analysis with the transcriptomic scores in both populations. We multiplied 
transcript loadings by measured gene expression in each mouse to generate 
a prediction of phenotype that captured the distally heritable portion of 
the transcriptome. The distributions 
of these scores across individuals were much more highly skewed than the 
genotype-informed scores. That is, obese mice had transcriptomes that were 
clearly skewed toward obesity, while lean mice had transcriptomes that were 
clearly skewed toward leanness. The mean of these scores (the polytranscriptomic
risk score - PtRS) explained between 43\% (skeletal muscle) 63\% (adipose tissue) 
of phenotypic outcome; substantially more than the TWAS-based PRS in the DO mice. 
Moreover, when applied to the CC-RIX transcriptomes, we were able to accurately 
predict body weight outcomes in this independent population, explaining up to
71\% of trait variance across individuals. The PtRS were furthmore able to predict 
obese/non-obese and diabetic/non-diabetic labels in human subjects with 
measured adipose transcription downloaded from the Gene Expression Omnibus 
(GEO). Together, these results suggest that the PRS developed in the DO 
performed well when predicting phenotype in the DO because it was capturing 
the distally heritable transcriptomic signature. It failed in the CC-RIX 
because it did not capture the distally heritable signature in this second 
population. Because it captures the distally heritable component of the 
transcriptome that is related to phenotype, the PtRS was able to predict 
phenotype from transcriptome data in the CC-RIX and in human transcriptome 
data. Thus, the polytranscriptomic signature we identified captured distally 
heritable transcriptomic information that is related to metabolic phenotypes 
across multiple mouse and human populations. Because the transcriptomic 
signature is gene-based, it can be functionally annotated. We demonstrate 
the adipose inflammation and macrophage infiltration is a heritable component 
of the development of obesity and insulin resistance in response to a high-fat, 
high-sugar diet. We further show that this transcriptional signature is 
targeted by the most common classes of drugs used in type 2 diabetes that 
are known to reverse insulin resistance.  

Such distal effects, rather than capturing the immediate,
local effects of genotype on gene expression, would capture broad genetic 
information, including massively polygenic effects and the effects of 
genetic interactions on all transcripts and phenotypes across developmental 
time. 


## Abstract/Results

We found that HDM successfully links genomes, transcriptomes, and phenomes in 
a mouse population with comprehensive data collection. 
We performed HDM in a population of 371 diversity outbred (DO) mice, 
both male and female, maintained on a high-fat, high-sugar diet. (DO mice are...) 
Individual genomes were determined with high-density genotyping chips and haplotype 
reconstruction. The mice were extensively phenotyped for multiple, clinically 
relevant metabolic measures including measures of obesity and insulin resistance. 
We measured transcriptome-wide gene expression in four tissues: white adipose 
tissue, pancreatic islets, liver, and skeletal muscle. We performed HDM using 
kernelized matrices of the genome, transcriptomes, and phenome to identify genome, 
transcriptome, and phenome scores where the transcriptome score perfectly mediated 
the effect of the genome score on the phenome score. We calculated loadings onto 
individual transcripts and phenotypes to interpret the scores. High phenome scores 
were associated with insulin resistance and high body weight, whereas low phenome 
scores were associated with insulin sensitivity and low body weight. Large positive 
transcript loadings indicated transcripts that were positively associated with 
phenome score. Large negative transcript loadings indicated transcripts that 
were negatively associated with phenome score. 

In assessing genotype-phenotype relationships, we found that local eQTL were 
insufficient for predicting trait variance in the validation population.
 We combined transcript loadings with local eQTL allele effects to identify 
 risk variants and protective variants in the DO mice. These annotations of 
 risk and protective alleles allowed us to perform a TWAS-like analysis within 
 the DO population as well as in an independent population of Collaborative Cross 
 (CC) F1 mice (CC-RIX). In the DO population, each mouse carried a relatively 
 balanced set of risk and protective alleles. The mean of these local allele 
 effects (the polygenic risk score - PGS) explained between 14\% (skeletal muscle 
 eQTL) and 47\% (adipose eQTL) of trait variance in the DO population, mirroring 
 success of TWAS within European populations. However, when we calculated PRS in 
 the CC-RIX based on the values derived in the DO population, PRS completely 
 failed to predict phenotype (numbers forthcoming). Although all the alleles 
 in the CC-RIX were all present in the DO population, the associations of risk 
 and protection as defined in the DO did not hold in the CC-RIX, supporting the 
 hypothesis that local eQTLs contribute minimally to phenotypic outcomes. 

In contrast, we discovered that the collective distal contributions to gene 
expression strongly predicted clinical traits in the validation population. 
We performed an analogous analysis with the transcriptomic scores in both 
populations. We multiplied transcript loadings by measured gene expression 
in each mouse to generate a prediction of phenotype that captured the distally 
heritable portion of the transcriptome. The distributions of these scores across 
individuals were much more highly skewed than the genotype-informed scores. 
Obese mice had transcriptomes that were clearly skewed toward positive scores, 
while lean mice had transcriptomes that were clearly skewed toward negative 
scores. The means of these scores (the polytranscriptomic risk score - PtRS) 
explained between 43\% (skeletal muscle) 63\% (adipose tissue) of phenotypic 
outcome in the DO mice--substantially more than PRS, which is based on local 
eQTLs. Moreover, when applied to the CC-RIX transcriptomes, PtRS accurately 
predict body weight outcomes in this independent population, explaining as much 
as 71\% of trait variance across individuals. PtRS was furthermore able to predict 
obese/non-obese and diabetic/non-diabetic labels in human subjects with measured 
adipose transcription downloaded from the Gene Expression Omnibus (GEO). 

Together, these results suggest that PRS based on local eQTLs performed well in 
the population in which it was developed because the eQTLs were densely sampled 
enough that they actually represent the distally heritable transcriptomic 
signature. PRS failed in the CC-RIX because it did not capture the distally 
heritable signature in this unrelated population. PtRS, on the other hand, was 
able to predict phenotype from transcriptome data in the DO, the CC-RIX, and in 
human transcriptome data. Thus PtRS captured distally heritable transcriptomic 
information that was related to metabolic phenotypes across multiple mouse and 
human populations. Furthermore, because the transcriptomic signature is gene-based, 
it can be functionally annotated. We demonstrated that adipose inflammation and 
macrophage infiltration were heritable components of the development of obesity 
and insulin resistance in response to a high-fat, high-sugar diet. We further 
showed that this transcriptional signature is targeted by the most common 
classes of drugs used in type 2 diabetes that are known to reverse insulin 
resistance.  HDM thus represents a promising method for the analysis of highly 
polygenic traits. It identifies biologically meaningful, interpretable, and 
translational transcriptomic signatures that mediate the effect of the genome 
on the phenome.
