% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{amsmath,amssymb}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math} % this also loads fontspec
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
\usepackage{lmodern}
\ifPDFTeX\else
  % xetex/luatex font selection
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering

\newcommand{\beginsupplement}{%
        \setcounter{table}{0}
        \renewcommand{\thetable}{S\arabic{table}}%
        \setcounter{figure}{0}
        \renewcommand{\thefigure}{S\arabic{figure}}%
     }

\usepackage{lineno}
\linenumbers

\usepackage{natbib}

\usepackage{setspace} \doublespacing
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same}
\hypersetup{
  pdftitle={Transcripts with high distal heritability mediate genetic effects on complex traits},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\title{Transcripts with high distal heritability mediate genetic effects
on complex traits}
\author{}
\date{\vspace{-2.5em}}

\begin{document}
\maketitle

\subsection{Abstract}\label{abstract}

The transcriptome is increasingly viewed as a bridge between genetic
risk factors for complex disease and their associated pathophysiology.
Powerful insights into disease mechanism can be made by linking genetic
variants affecting gene expression (expression quantiatitive trait loci
- eQTLs) to phenotypes.

\subsection{Introduction}\label{introduction}

In the quest to understand genetic contributions to complex traits, gene
expression is an important bridge between genotype and phenotype. By
identifying transcripts that mediate the effect of genetic loci on
traits, we get one step closer to a mechanistic understanding of the
influence of genetic variants on traits. There is evidence from
genome-wide association studies (GWAS) that regulation of gene
expression accounts for the bulk of the genetic effect on complex
traits, as most trait-associated variants lie in gene regulatory regions
\cite{pmid22955828, pmid25363779, pmid21617055, pmid19474294, 
pmid24702953, pmid24316577, pmid27126046}. It is widely assumed that
these variants influence local transcription, and methods such as
transcription-wide association studies (TWAS)
\cite{pmid26258848, pmid27019110, pmid26854917} summary data-based
Mendelian randomization (SMR) \cite{pmid27019110}, and others {[}cite{]}
have capitalized on this idea to identify genes associated with multiple
disease traits {[}cite many{]}

Despite the great promise of these methods, however, they have not been
as widely successful as it seemed they could have been, and the vast
majority of complex trait heritability remains unexplained. Although
trait-associated variants tend to lie in non-coding, putative regulatory
regions, they often do not have detectable effects on gene expression
\cite{pmid32912663} and do not tend to co-localize with eQTL
\cite{pmid36515579, 
pmid37857933}.

One possible explanation for these observations is that we are not
measuring gene expression in the appropriate cell types and are thus
unable to detect true eQTLs influencing traits \cite{pmid32912663}. An
alternative explanation that has been discussed in recent years is that
heritability of these variants is mediated not through local regulation
of gene expression, but through distal regulation
\cite{pmid37857933, pmid32424349}. Yao \textit{et al.}
\cite{pmid32424349} observed that genes with low local heritability
explain more expression-mediated disease heritability than genes with
high local heritability. This observation is consistent with principles
of robustness in complex systems. If a transcript were both important to
a trait and subject to strong local regulation, a population would be
susceptible to extremes in phenotype that could be defined as disease.
Indeed, strong disruption of trait-important genes is the cause of
Mendelian disease. Instead, what has been observed is that genes that
are near GWAS hits and have obvious functional relevance to a trait tend
to have highly complex regulatory landscapes under strong selection
pressures \cite{pmid37857933}. In contrast, genes with strong local
regulation tend to be depleted of functional annotations and are under
looser selection constraints \cite{pmid37857933}. These observations and
others led Liu et al.~\cite{pmid31051098} to suggest that most
heritability of complex traits is driven by weak trans-eQTLs. They
proposed a framework of understanding heritability of complex traits in
which massive polygenicity is distributed across common variants in both
functional core genes, as well as more peripheral genes that may not
seem obviously related to the trait.

If this model is correct, it should be the case that\ldots{}

With dense enough data, it should be possible to test this model.

Here we investigated the roles of local and distal gene regulation on
the heritability of complex metabolic traits in a population of
diversity outbred (DO) mice. DO mice are derived from eight inbred
founder mouse strains, five classical lab strains, and three strains
more recently derived from wild mice \cite{pmid22892839}. They represent
three subspecies of mouse \textit{Mus musculus domesticus},
\textit{Mus musculus musculus}, and \textit{Mus musculus casteneus}, and
capture 90\% of the known variation in laboratory mice {[}cite{]}. We
placed a population of 500 mice, both male and female, on a high-fat,
high-sugar diet \cite{pmid29567659} to induce diet-associated obesity
and metabolic disease. Over 18 weeks multiple metabolic traits were
measured, including body weight, plasma levels of insulin and glucose,
and plasma lipids. At the end of the experiment, we used RNASeq to
measure gene expression in 384 mice in four tissues relevant to
metabolic disease: adipose tissue, pancreatic islets, liver, and
skeletal muscle. The mice were also genotyped using the Mouse Universal
Genotyping Array (GigaMUGA). The high dimensionality of phenotyping,
genotyping, and transcriptome measurements in this large number of
genetically diverse animals enabled thorough inquiry into the role of
local and distal eQTL in the heritability of complex traits. Further,
measuring gene expression in multiple tissues, enabled us to assess the
importance of measuring gene expression in appropriate cell types.

To investigate the extent to which local and distal gene regulation
across multiple tissues mediate genetic effects on complex traits, we
propose a systems approach called high-dimensional mediation (HDM). HDM
uses a regularized and generalized canonical correlation analysis
(RGCCA) {[}cite{]}, which is an extension of canonical correlation
analysis (CCA) that allows for more than two data sets with arbitrary
relationships among them. Thus, we can identify linear combinations of
the genome, transcriptome, and phenome, that describe the mediation of
the genetic effects on the phenome through the transcriptome. This
systems approach, as opposed to a univariate approach, puts local and
distal regulation on the same statistical level

Because of the central dogma of molecular biology, information flow is
directed out of the genome, and not back into it. Thus, the otherwise
undirected relationships between genome, transcriptome, and phenome can
be inferred as a causal mediation by the transcriptome of the effects of
the genome on the phenome.

\subsection{Results}\label{results}

\subsubsection{Genetic variation contributes to wide phenotypic
variation}\label{genetic-variation-contributes-to-wide-phenotypic-variation}

A population of 500 diversity outbred mice (split evenly between male
and female) from generates 18, 19, and 21, was placed on a high-fat
(44.6\% kcal fat), high-sugar (34\% carbohydrate), adequate protein
(17.3 \% protein) diet from Envigo Teklad (catalog number TD.08811)
starting at four weeks of age as described previously
\cite{pmid29567659}.

Each individual was assessed longitudinally for multiple metabolic
measures including fasting glucose levels, glucose tolerance, insulin
levels, body weight, and blood lipid levels (Methods).

Although the environment was consistent across animals, the genetic
diversity present in this population resulted in widely varying
distributions across physiological measurements (Fig.
\ref{fig:trait_overview}). For example, body weights of adult
individuals varied from less than the average adult B6 body weight to
several times the body weight of a B6 adult in both sexes (Fig.
\ref{fig:trait_overview}A). Fasting blood glucose (FBG) also varied
considerably (Fig. \ref{fig:trait_overview}B) although few of the
animals had FBG levels that would indicate pre-diabetes ( animals, ), or
diabetes (7 animals, 1.4) according to previously developed cutoffs
(pre-diabetes: FBG \(\geq\) 250 mg/dL, diabetes: FBG \(\geq\) 300,
mg/dL) \cite{pmid17018838}. Males had higher FBG than females on average
(Fig. \ref{fig:trait_overview}C) as has been observed before suggesting
either that males were more susceptible to metabolic disease on the
high-fat diet, or that males and females may require different
thresholds for pre-diabetes and diabetes.

Body weight was strongly positively correlated with food consumption
(Fig. \ref{fig:trait_overview}D \(R^2 =\) 0.51, \(p=\)
\ensuremath{1.5\times 10^{-75}}) and fasting blood glucose (FBG) (Fig.
\ref{fig:trait_overview}E, \(R^2=\) 0.25, \(p =\)
\ensuremath{2\times 10^{-32}}) suggesting a link between behavioral
factors and metabolic disease. However, the heritability of this trait
and others (Fig. \ref{fig:trait_overview}F) indicates that background
genetics contribute substantially to correlates of metabolic disease in
this population.

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig1_trait_overview.pdf} 
\caption{Clinical overview. \textbf{A.} Distributions of final body weight 
in the diversity outbred mice. Sex is indicated by color. The 
average B6 male and female adult weights at 24 weeks of age 
are indicated by blue and green bars on the x-axis. \textbf{B.} The 
distribution of final fasting glucose across the population split 
by sex. Normal, pre-diabetic, and diabetic fasting glucose levels 
for mice are shown by colored bars along the x-axis. \textbf{C.} Males had 
higher fasting blood glucose on average than females. \textbf{D.} The 
relationship between food consumption and body weight for both 
sexes. \textbf{E.} Relationship between body weight and fasting glucose 
for both sexes. \textbf{F.} Heritability estimates for each physiological 
trait. Bars show standard error of the estimate. \textbf{G.} Correlation 
structure between pairs of physiological traits.}
\label{fig:trait_overview}
\end{figure}

The landscape of trait correlations (Fig. \ref{fig:trait_overview}G)
shows that most of the metabolic trait pairs were relatively weakly
correlated indicating complex relationships among the measured traits.
This low level of redundancy suggests a broad sampling of multiple
heritable aspects of metabolic disease including overall body weight,
glucose homeostasis, pancreatic composition and liver function.

\subsubsection{Distal Heritability Correlates with Phenotype
Relevance}\label{distal-heritability-correlates-with-phenotype-relevance}

To elaborate the mechanistic details of genetic effects on metabolic
phenotypes in the DO population, we also measured gene expression in
four tissues known to be involved in metabolic disease: adipose,
pancreatic islet, liver, and skeletal muscle. To confirm the
heritability of transcript levels, we performed expression QTL analysis
using R/qtl2 {[}cite{]} (Methods) and identified both local and distal
eQTL for transcripts in each tissue (Supp. Fig \ref{fig:eQTL}).
Significant local eQTLs far outnumbered distal eQTLs (Supp. Fig.
\ref{fig:eQTL}F) and tended to be shared across tissues (Supp. Fig.
\ref{fig:eQTL}G) whereas the few significant distal eQTL we identified
tended to be tissue-specific (Supp. Fig. \ref{fig:eQTL}H)

To better compare the relative contribution of local and distal genetics
to transcript levels, we performed a heritability analysis for each
transcript (Methods). Overall, local and distal factors contributed
approximately equally to transcript abundance. In all tissues, both
local and distal factors explained between 13 and 19\% of the variance
in the median transcript (Fig \ref{fig:motivation}A).

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig2_motivation.pdf} 
\caption{Transcript heritability and trait relevance. 
\textbf{A.} Distributions of distal and local heritability of 
transcripts across the four tissues. Overall local and distal 
factors contribute equally to transcript heritability. The 
relationship between (\textbf{B.}) local and (\textbf{C.}) 
distal heritability and trait relevance across all four tissues. 
Here trait relevance is defined as the maximum correlation between 
the transcript and all traits. Local heritability is negatively 
correlated with trait relevance, and distal heritability is 
positively correlated with trait relevance. Pearson ($r$) and $p$ 
values for each correlation are shown in the upper-right of each panel.}
\label{fig:motivation}
\end{figure}

Local heritability of transcripts was negatively correlated with their
trait relevance, defined as the maximum correlation of a transcript
across all traits (Fig. \ref{fig:motivation}B). This suggests that the
more local genotype influenced transcript abundance, the less effect
variation in transcript abundance was related to the measured traits.
Conversely, distal heritability of transcripts was positively correlated
with trait relevance (Fig. \ref{fig:motivation}C). That is, transcripts
that were more highly correlated with the measured traits tended to be
distally, rather than locally, heritable. That trait-correlated
transcripts have low local heritability is consistent with previous
observations that low-heritability transcripts explain more
expression-mediated disease heritability than high-heritability
transcripts \cite{pmid32424349}. However, the positive relationship
between trait correlation and distal heritability suggests that there
are alternative mechanisms through which genetic regulation of
transcripts may influence traits.

\subsubsection{High-Dimensional Mediation identifies composite
transcript that perfectly mediates composite
trait}\label{high-dimensional-mediation-identifies-composite-transcript-that-perfectly-mediates-composite-trait}

To identify mechanisms through which genetic regulation of transcripts
influences heritable traits, we propose high-dimensional mediation (HDM)
(Fig. \ref{fig:workflow}). In this process we kernelize each of the
genome, transcriptome, and phenome, and perform regularized and sparse
generalized canonical correlation analysis (RGCCA) {[}cite{]} in which
we explicitly model the mediation by the transcriptome of the effect of
the genome on the phenome (Methods, Fig. \ref{fig:workflow}). RGCCA is
an extended form of canonical correlation analysis (CCA) {[}cite{]} in
which multiple data sets can be analyzed simultaneously with explicit
relationships.

\begin{figure}[ht!]
\includegraphics[width=5in]{Figures/Fig3_workflow.pdf} 
\caption{High-dimensional mediation. \textbf{A.} Workflow indicating 
major steps of high-dimensional mediation. The genotype, transcriptome, 
and phenotype matrices were kernelized to yield single matrices representing 
the relationships between all individuals for each data modality ($G_K$ = 
genome kernel, $T_K$ = transcriptome kernel; $P_K$ = phenome kernel). High-dimensional 
mediation was applied to these matrices to maximize the direct path 
$G \rightarrow T \rightarrow P$, the mediating pathway (arrows), while 
simultaneously minimizing the direct $G \rightarrow P$ pathway (dotted line). 
The composite vectors that resulted from high-dimensional mediation were 
$G_c$, $T_C$, and $P_C$. The partial correlations $\rho$ between these vectors 
indicated perfect mediation. Transcript and trait loadings were calculated 
as described in the methods. \textbf{B.} The null distribution of the path 
coefficient derived from 10,000 permutations compared to the observed path 
coefficient (red line). \textbf{C.} The null distribution of the $G_C$-$T_C$ 
correlation vs. the $T_C$-$P_C$ correlation compared with the observed value 
(red dot).
}
\label{fig:workflow}
\end{figure}

The result of this process is three vectors representing the composite
genome (\(G_C\)), composite transcriptome (\(T_C\)) and the composite
phenome (\(P_C\)) where the composite transcriptome perfectly mediates
the effect of the composite genome on the composite phenome. Each vector
is of length \(n\) where \(n\) is the number of individual mice. Fig.
\ref{fig:workflow}A shows the partial correlations between all pairs of
composite vectors. The partial correlation \(r\) between \(G_C\) and
\(T_S\) was 0.46, and the partial correlation between \(T_S\) and
\(P_S\) was 0.78. However, when the transcriptome was taken into
account, the partial correlation between \(G_S\) and \(P_S\) was
effectively 0 (-0.01).

Standard CCA is prone to over-fitting because in any two large matrices
it can be trivial to identify highly correlated composite vectors. To
assess whether RGCCA was similarly prone to over-fitting in a
high-dimensional space, we performed permutation testing. We permuted
the individual labels on the transcriptome kernel matrix 1000 times and
recalculated the path coefficient, which is the partial correlation of
\(G_C\) and \(T_C\) multiplied by the partial correlation of \(T_C\) and
\(P_C\). This represents the path from \(G_C\) to \(P_C\) that is
mediated through \(T_C\). The null distribution of the path coefficient
is shown in Fig. \ref{fig:workflow}B, and the observed path coefficient
from the original data is indicated by the red line. The observed path
coefficient was well outside the null distribution generated by
permutations. Fig. \ref{fig:workflow}C illustrates this observation in
more detail. Although we identified high correlations between \(G_C\)
and \(T_C\), and modest correlations between \(T_C\) and \(P_C\) in the
null data (Fig \ref{fig:workflow}C), these two values could not be
maximized simultaneously. The red dot shows that in the real data both
the \(G_C\)-\(T_C\) correlation and the \(T_C\)-\(P_C\) correlation
could be maximized simultaneously suggesting that that path from
genotype to phenotype through transcriptome is highly non-trivial and
identifiable in this case. These results suggest that these composite
vectors represent genetically determined variation in phenotype that is
mediated through genetically determined variation in transcription.

\subsubsection{Body weight and insulin resistance were highly
represented in the expression-mediated composite
trait}\label{body-weight-and-insulin-resistance-were-highly-represented-in-the-expression-mediated-composite-trait}

The loadings of each measured trait onto \(P_C\) indicate how much each
contributed to \(P_C\). Final body weight contributed the most to
\(P_C\) (Fig. \ref{fig:interpretation}), followed by homeostatic insulin
resistance (HOMA\_IR) and fasting plasma insulin levels
(Insulin\_Fasting). We can thus interpret \(P_C\) as an index of
metabolic disease (Fig. \ref{fig:interpretation}B). Individuals with
high values of \(P_C\) have a higher metabolic index and greater
metabolic disease, including higher body weight and higher insulin
resistance. We refer to \(P_C\) as the metabolic index going forward.
Traits contributing the least to the metabolic index were measures of
cholesterol and pancreas composition. Thus, when we interpret the
transcriptomic signature identified by HDM, we are explaining primarily
transcriptional mediation of body weight and insulin resistance, as
opposed to cholesterol measurements.

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig4_interpretation.pdf} 
\caption{Interpretation of loadings. \textbf{A.} Loadings 
across traits. Body weight and insulin resistance contributed 
the most to the composite trait. \textbf{B.} Phenotype scores 
across individuals. Individuals with large positive phenotype 
scores had higher body weight and insulin resistance than average. 
Individuals with large negative phenotype scores had lower body 
weight and insulin resistance than average. \textbf{C.} 
Distribution of transcript loadings in adipose tissue. For 
transcripts with large positive loadings, higher expression was 
associated with higher phenotype scores. For transcripts with 
large negative loadings, higher expression was associated with 
lower phenotype scores. \textbf{D.} Distribution of absolute 
value of transcript loadings across tissues. Transcripts in 
adipose tissue had the largest loadings indicating that 
transcripts in adipose tissue were the best mediators of the 
genetic effects on body weight and insulin resistance.
}
\label{fig:interpretation}
\end{figure}

\subsubsection{High-loading transcripts have low local heritability,
high distal heritability, and are linked mechanistically to
obesity}\label{high-loading-transcripts-have-low-local-heritability-high-distal-heritability-and-are-linked-mechanistically-to-obesity}

We interpreted large loadings onto transcripts as indicating strong
mediation of the effect of genetics on metabolic index. Large positive
loadings indicate that inheriting higher expression was associated with
a higher metabolic index (i.e.~higher risk of obesity and metabolic
disease on the high-fat diet) (Fig. \ref{fig:interpretation}C).
Conversely, large negative loadings indicate that inheriting lower
expression of these transcripts was associated with a lower metabolic
index (i.e.~lower risk of obesity and metabolic disease on the high-fat
diet) (Fig. \ref{fig:interpretation}C). We used GSEA to look for
biological processes and pathways that were enriched at the top and
bottom of this list (Methods).

In adipose tissue, both GO processes and and KEGG pathway enrichments
pointed to an axis of inflammation and metabolism (Supp. Fig.
\ref{fig:top_enrich_kegg} and \ref{fig:top_enrich_go}). Processes and
pathways associated with inflammation, particularly macrophage
infiltration were positively associated with metabolic index, indicating
that increased expression in inflammatory pathways was associated with a
higher metabolic index. It is well established that adipose tissue in
obese individuals is highly inflamed {[}cite{]} and infiltrated by
macrophages {[}cite{]}, and the results here suggest that this may be a
heritable component of metabolic disease.

The strongest negative enrichments in adipose tissue were related to
mitochondial activity in general, and thermogenesis in particular (Supp.
Fig. \ref{fig:top_enrich_kegg} and \ref{fig:top_enrich_go}). It has been
shown mouse strains with greater thermogenic potential are also less
susceptible to obesity on a high-fat diet.

Transcripts associated with the citric acid (TCA) cycle as well as the
catabolism of branched-chain amino acids (BCAA), valine, leuceine, and
isoleucine also had strong negative enrichment in the adipose tissue
(Supp. Fig. XXX). Expression of genes in both pathways (for which there
is some overlap) has been previously associated with insulin sensitivity
\cite{pmid29567659,pmid22560213,pmid19841271}, suggesting that
impairment in these pathways may be associated with insulin resistance.
Selective PPAR\(\gamma\) modulation by insulin-sensitizing
thiazolidinedione drugs has further been shown to influence both
inflammation and BCAA metabolism in obese rats suggesting a relationship
between these pathways and insulin resistance \cite{pmid20959535}. BCAA
levels are also related to insulin resistance in human subjects and are
elevated in insulin-resistant obese individuals relative to
weight-matched non-insulin reisistant individuals \cite{pmid23512805}.
In the DO mice studied here, inheriting increased expression of genes
involved in BCAA catabolism was associated with reduced body weight and
insulin resistance.

Transcripts in the adipose tissue had the largest loadings, both
positive and negative, of all tissues, suggesting that much of the
effect of genetics on body weight and insulin reisistance is mediated
through gene expression in adipose tissue (Fig.
\ref{fig:loading_heritability}A). The loadings in liver and pancreas
were comparable, and those in skeletal muscle were the weakest (Fig.
\ref{fig:loading_heritability}A), suggesting that less of the genetic
effects were mediated through transcription in skeletal muscle. Across
all tissues, trahscripts with the largest loadings tended to have
relatively high distal heritability compared with local heritability
(Fig. \ref{fig:loading_heritability}A). Transcripts with the highest
local heritability tended to have very weak loadings and were 3.6 times
less likely to be associated with diabetes and obesity in the literature
than transcripts with high loadings (Fig. fig:loading\_heritabilityB,
Methods). TWAS-nominated transcripts also had relatively weak loadings
and high local heritabilty (Fig. \ref{fig:interpretation}C). They were
half as likely as transcripts with the highest loadings to be associated
with diabetes and obesity in the literature (Fig.
\ref{fig:interpretation}C).

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig5_loading_heritability.pdf} 
\caption{Transcripts with high loadings have high distal heritability
and literature support. Each panel has a bar plot showing the loadings 
of transcripts selected by different criteria. Bar color indicates the 
tissue of origin. The heat map shows the local (L - left) and distal 
(D - right) heritability of each transcript. \textbf{A.} Loadings for 
the 10 transcripts with the largest positive loadings and the 10 
transcripts with the largest negative loadings for each tissue. 
\textbf{B.} Loadings of TWAS candidates with the 10 largest positive 
correlations with traits and the largest negative correlations with 
traits across all four tissues. \textbf{C.} The transcripts with the 
largest local heritability (top 20) across all four tissues.
}
\label{fig:loading_heritability}
\end{figure}

\subsubsection{Tissue-specific transriptional programs are associated
with metabolic
traits}\label{tissue-specific-transriptional-programs-are-associated-with-metabolic-traits}

Clustering of transcripts with top loadings in each tissue shows
tissue-specific functional modules associated with obesity and insulin
resistance in the DO population (Fig. \ref{fig:toa}A). In this figure,
the importance of immune activation specifically in the adipose tissue
is apparent. There are also other tissue-specific processes. Positive
loadings on lipid metabolism in liver suggest that inheriting high liver
expression of genes in this cluster is positively associated with
metabolic disease. This cluster included the gene \textit{Pparg}, whose
primary role is in the adipose tissue where it is considered a master
regulator of adipogenesis \cite{pmid17389767}. Agonists of \text{Pparg},
such as Thiazolidinediones, which are FDA-approved to treat type II
diabetes, reduce inflammation and adipose hyptertrophy
\cite{pmid17389767}. Consistent with this role, the loading for
\textit{Pparg} in adipose tissue is slightly negative, suggesting that
upregulation is associated with leaner mice (Fig. \ref{fig:toa}B). In
contrast, \textit{Pparg} has a large positive loading in liver, where it
plays a role in the development of hepatic steatosis, or fatty liver.
Mice that lack \textit{Pparg} specifically in the liver, are protected
from developing steatosis and show reduced expression of lipogenic genes
\cite{pmid12805374, pmid12618528}. Overexpression of \textit{Pparg} in
the livers of mice with a \textit{Ppara} knockout, causes upregulation
of genes involved in adipogenesis \cite{pmid16357043}. In the livers of
both mice and humans \cite{pmid15644454, pmid16403437} High
\textit{Pparg} expression is associated with hepatocytes that accumulate
large lipid droplets and have gene expression profiles similar to
adipocytes.

The local and distal heritability of \textit{Pparg} is low in adipose
tissue suggesting its expression in this tissue is highly constrained in
the population (Fig. \ref{fig:toa}B). However, the distal heritability
of \textit{Pparg} in liver is relatively high suggesting it is complexly
regulated and has sufficient variation in this population to drive
variation in phenotype.

\begin{figure}[ht!]
\includegraphics[width=4in]{Figures/Fig6_TOA.pdf} 
\caption{Tissue-specific transcriptional programs are associated 
with obesity and insulin resistance. \textbf{A} Heat map showing 
the loadings of all transcripts with loadings greater than 2.5 
standard deviations from the mean in any tissue. The heat map was 
clustered using k medoid clustering. Functional enrichments of each 
cluster are indicated along the left margin. \textbf{B} Loadings for 
\textit{Pparg} in different tissues. \textbf{C} Local and distal of 
\textit{Pparg} expression in different tissues.
}
\label{fig:toa}
\end{figure}

\subsubsection{Gene expression, but not local eQTLs predict body weight
in an independent
population}\label{gene-expression-but-not-local-eqtls-predict-body-weight-in-an-independent-population}

The loading of each transcript indicates how inherited expression levels
influence metabolic phenotypes. If local regulation is the predominant
factor influencing gene expression, we should be able to predict an
individual's phenotype based on their genotypes across all local eQTLs.
We tested this hypothesis in an independent population of F1 mice
generated through multiple pairings of Collaborative Cross (CC)
{[}cite{]} strains (Fig. \ref{fig:cc_prediction}A) (Methods).

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig7_CC_Prediction.pdf} 
\caption{Transcription, but not local genotype, predicts 
phenotype in the CC-RIX. \textbf{A.} Workflow showing procedure 
for translating HDM results to an independent population of mice. 
\textbf{B.} Relationships between the predicted metabolic index 
and measured body weight. The left column shows the predictions 
using measured transcripts. The right column shows the prediction 
using transcript levels imputed from local genotype. Gray boxes 
indicate measured quantities, and blue boxes indicate calculated 
quantities. The dots in each panel represent individual CC-RIX strains. 
The gray lines show the standard deviation on body weight for the strain.
}
\label{fig:cc_prediction}
\end{figure}

We first tested whether the transcript loadings derived from HDM in the
DO were relevant to the relationship between the transcriptome and the
phenome in the CC-RIX. To do this, we multiplied the transcript loadings
derived from HDM in the DO mice by transcript measurements in the CC-RIX
standardized across individuals. This created a transcript vector
weighted by importance to metabolic disease as determined in the DO. The
mean of this vector was the predicted metabolic index for the animal
based on its transcription in either adipose tissue, liver, or skeletal
muscle. Across all three tissues, weighted transcription values were
significantly correlated with metabolic index in the CC-RIX population
measured as body weight (Fig. \ref{fig:cc_prediction}B left column).
Adipose tissue transcription yielded the most accurate prediction
(stats). This result confirms the validity and translatability of the
transcript loadings determined in the DO population and their
relationship to metabolic disease. It also supports the observation that
transcription in adipose tissue is the strongest mediator of genetic
effects on metabolic index.

We then tested whether this mediation signal was encoded by local
genotype. To do this, we imputed gene expression in the CC-RIX using
local genotype. We were able to estimate variation in gene transcription
robustly. The correlation between measured gene expression and imputed
gene expression across all tissues was close to \(R = 0.5\), and the
variance explained by local genotype was comparable in the DO and CC-RIX
(Supp. Fig. \ref{fig:cc_imputation}). However, when weighted with the
loadings derived from HDM in the DO population, these imputed
transcripts across all tissues failed to predict metabolic index in the
CC-RIX (Fig. \ref{fig:cc_prediction}B right column).

Taken together, these results support the hypothesis that distal, rather
than local genetic factors are primarily driving complex-trait related
variation in gene expression.

\subsubsection{Distally heritable transcriptomic signatures reflect
variation in composition of adipose tissue and
islets}\label{distally-heritable-transcriptomic-signatures-reflect-variation-in-composition-of-adipose-tissue-and-islets}

Interpretation of global distal genetic influences on gene expression
and phenotype is potentially more challenging than interpretation and
translation of local genetic influences. Effects can not be located to
individual gene variants or transcripts, but because we have a measure
of importance across all transcripts in multiple tissues, we can look at
global patterns. We noted earlier that functional enrichments of
transcripts with large positive loadings in the adipose tissue,
suggested that the obese mice in the population had a genetic
predisposition toward elevated macrophage infiltration into the adipose
tissue. This suggests heritabl variability in cell-type composition of
the adipose tissue. We investigated this further bioinformatically by
comparing the loadings of cell-type-specific transcripts (Methods). For
adipose tissue we used a list of cell-type specific genes identified in
human adipose tissue

In adipose tissue, the mean loading of macrophage-specific genes was
substantially greater than 0 (Fig. \ref{fig:human_translation}A),
indicating that obese mice were genetically predisposed to have high
levels of macrophage infiltration in adipose tissue in response to the
high-fat, high-sugar diet.

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Fig8_Human_Translation.pdf} 
\caption{HDM results translate to humans. \textbf{A.} Distribution of 
loadings for cell-type-specific transcripts in adipose tissue. \textbf{B.} 
Distribution of loadings for cell-type-specific transcripts in pancreatic 
islets (green). \textbf{C.} Null distributions for the mean loading of 
randomly selected transcripts in each cell type compared with the observed 
mean loading of each group of transcripts (red asterisk). \textbf{D.} 
Predictions of metabolic phenotypes in four adipose transcription data 
sets downloaded from GEO. In each study the obese/diabetic patients were 
predicted to have greater metabolic disease than the lean/non-diabetic 
patients based on the HDM results from DO mice.
}
\label{fig:human_translation}
\end{figure}

In islet, the mean loadings for alpha-cell specific transcripts were
significantly positive, while the mean loadings for delta- and
endothelial-cell specific genes were significantly negative (Fig.
\ref{fig:human_translation}B). These results suggest that obese mice had
inherited higher proportions of alpha cells, and lower proportions of
endothelial and delta cells in their pancreatic islets.

The loadings for pancreatic beta cell-type specific loadings was not
significantly different from zero. This does not reflect on the function
of the beta cells in the obese mice, but rather suggests that mice prone
to obesity were not obese because they inherited fewer beta cells than
non-obese mice.

Biological interpretation of alpha, endothelial, delta cells??

\subsubsection{Distally heritable transcriptomic signatures translate to
human
disease}\label{distally-heritable-transcriptomic-signatures-translate-to-human-disease}

Ultimately, the distally heritable transcriptomic signatures that we
identified in DO mice will be useful if they inform pathogenicity and
treatment of human disease. To investigate the potential for translation
of the gene signatures identified in DO mice, we compared them to
transcriptional profiles in obese and non-obese human subjects
(Methods). We limited our analysis to adipose tissue because the adipose
tissue signature had the strongest relationship to obesity and insulin
resistance in the DO.

We calculated a predicted obesity score for each individual in the human
studies based on their adipose tissue gene expression (Methods) and
compared the predicted scores for obese and non-obese groups as well as
diabetic and non-diabetic groups. In all cases, the predicted obesity
scores were higher on average for individuals in the obese and diabetic
groups compared with the lean and non-diabetic groups, indicating that
the distally heritable signature of obesity identified in DO mice is
relevant to obesity and diabetes in human subjects.

\subsubsection{Targeting gene
signatures}\label{targeting-gene-signatures}

Although high-loading transcripts are likely good candidates for
understanding specific biology related to obesity, we emphasize that the
transcriptome overall is highly interconnected and redundant, and that
focusing on individual transcripts for treatment may be less effective
than using the transcriptomic signature as a whole. The ConnectivityMap
(CMAP) database {[}cite{]} developed by the Broad Institute allows us to
query thousands of compounds that reverse or enhance transcriptomic
signatures as a whole in multiple different cell types. By identifying
drugs that reverse pathogenic transcriptomic signatures as a whole
rather than targeting individual genes, we can potentially increase
efficacy of tested compounds.

We thus queried the CMAP database through the CLUE online query tool
developed by The Broad Institute {[}cite{]} (Methods).

Alternatively, we can target the gene signature as a whole using CMAP.
Identifying drugs to target gene signatures is possible through CMAP. We
put our loadings from islet into CMAP. The top hit was PPAR receptor
agonist. Rosiglitazone, a widely used diabetes drug, is a PPAR receptor
agonist. Another class of drugs on the list was sufonylureas, which are
another major class of drugs for type 2 diabetes.

\begin{itemize}
\tightlist
\item
  \textbf{Supplemental Table} results from CMAP
\end{itemize}

\subsection{Discussion}\label{discussion}

\begin{itemize}
\item
  distal heritability correlates with phenotype relevance
\item
  others who use local eQTL to associate genotype with traits often say
  ``we nominated this gene'' even though other nearby genes have higher
  eQTL LOD scores (27019110, 31465442) Our method supports the idea that
  the transcripts with the strongest local regulation are less likely to
  be functionally related to the trait
\end{itemize}

\subsection{Data Availability}\label{data-availability}

Here we tell people where to find the data

\subsection{Acknowledgements}\label{acknowledgements}

Here we thank people

\pagebreak

\subsection{Supplemental Figures}\label{supplemental-figures}

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Supp_Fig1_eQTL.pdf} 
\caption{Overview of eQTL analysis in DO mice. \textbf{A.} RNA seq 
samples from the four different tissues clustered by tissue. 
\textbf{B.-E.} eQTL maps are shown for each tissue. The $x$-axis 
shows the position of the mapped eQTL, and the $y$-axis shows the 
physical position of the gene encoding each mapped transcript. 
Each dot represents an eQTL with a minimum LOD score of 8. The dots 
on the diagonal are locally regulated eQTL for which the mapped eQTL 
is at the within 4Mb of the encoding gene. Dots off the diagonal are 
distally regulated eQTL for which the mapped eQTL is distant from the 
gene encoding the transcript. \textbf{F.} Comparison of the total number 
of local and distal eQTL with a minimum LOD score of 8 in each tissue. 
All tissues have comparable numbers of eQTL. Local eQTL are much more 
numerous than distal eQTL. \textbf{G.} Counts of transcripts with local 
eQTL shared across multiple tissues. The majority of local eQTL were 
shared across all four tissues. \textbf{H.} Counts of transcripts with 
distal eQTL shared across multiple tissues. The majority of distal eQTL 
were tissue-specific and not shared across multiple tissues. For both G 
and H, eQTL for a given transcript were considered shared in two tissues 
if they were within 4Mb of each other. Colored bars indicate the counts 
for individual tissues for easy of visualization.
}
\label{fig:eQTL}
\end{figure}

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Supp_Fig_enrichments_KEGG.pdf} 
\caption{Bar plots showing normalized enrichment scores (NES) for KEGG 
pathways as determined by fast gene score enrichment analysis (fgsea). 
Only the top 10 positive and top 10 negative scores are shown. Colors 
indicate tissue. The name beside each bar shows the name of each enriched 
KEGG pathway.
}
\label{fig:top_enrich_kegg}
\end{figure}

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Supp_Fig_enrichments_GO.pdf} 
\caption{Bar plots showing normalized enrichment scores (NES) for GO 
terms as determined by fast gene score enrichment analysis (fgsea). 
Only the top 10 positive and top 10 negative scores are shown. Colors 
indicate tissue. The name beside each bar shows the name of each enriched 
GO term. The letters in parentheses indicate whether the term is from the 
biological process ontology (BP), the molecular function ontology (MF), 
or the cellular compartment ontology (CC).
}
\label{fig:top_enrich_go}
\end{figure}

\begin{figure}[ht!]
\includegraphics[width=\textwidth]{Figures/Supp_Fig_CC-RIX_Imputation.pdf} 
\caption{Validation of transcript imputation in the CC-RIX. \textbf{A.} 
Distributions of correlations between imputed and measured transcripts 
in the CC-RIX. The mean of each distribution is shown by the red line. 
All distributions were skewed toward positive correlations and had
 positive means near a Pearson correlation (r) of 0.5. \textbf{B.} 
 The relationship between the correlation between measured and 
 imputed expression in the CC-RIX (x-axis) and eQTL LOD score. As 
 expected, imputations are more accurate for transcripts with strong 
 local eQTL. \textbf{C.} Variance explained by local genotype in the 
 DO and CC-RIX. 
}
\label{fig:cc_imputation}
\end{figure}

\pagebreak

\bibliographystyle{unsrt}
\bibliography{islet.bib}

\end{document}
